packages/host/src/lib/host.ts

```diff
@@ -230,7 +231,7 @@ export class Host implements IComponent {
             this.logger.error("Setting telemetry failed");
         });
 
-        this.telemetryAdapter?.push("info", { message: "Host started", labels: { hostSize: this.hostSize } });
+        this.telemetryAdapter?.push("info", { message: "Host started", labels: { hostSize: this.hostSize, ip: this.ipvAddress} });
```

Comment:
Could we please add also a sequence adapter? It should be easy since the sequence adapter information is read a bit lower at:
`const adapter = await initializeSequenceAdapter(this.config);`
Then we can move the Telemetry push for "Host started" just after it.